jbmc/src/java_bytecode/java_bytecode_internal_additions.cpp

```diff
@@ -58,4 +74,6 @@ void java_internal_additions(symbol_table_baset &dest)
     symbol.is_static_lifetime = true;
     dest.add(symbol);
   }
+
+  create_java_initialize(dest);
```

Comment:
I wouldn't hide this inside `internal_additions`, just explicitly call it where needed

Reply:
Done.